_maps/configs/independent_mudskipper.json

```diff
@@ -29,7 +29,7 @@
 		},
 		"Salvage Technician": {
 			"outfit": "/datum/outfit/job/independent/engineer",
-			"slots": 1
+			"slots": 2
```

Comment:
This... Isn't increasing the limit. This is adding a second engi.

Reply:
I'm very tired.